locales/en/fields.json

```diff
@@ -259,11 +259,9 @@
   },
   "sentence_length": {
     "label": "Sentence length",
-    "items": {
```

Comment:
Any reason to remove the `items` key? It's something we've used consistently elsewhere to show the keys for items of a field.